website/source/docs/agent/config_entries.html.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: "docs"
+page_title: "Configuration Entry Definitions"
+sidebar_current: "docs-agent-config-entries"
+description: |-
+  Consul allows storing configuration entries centrally to be used as defaults for configuring other aspects of Consul.
+---
+
+# Configuration Entries
+
+Configuration entries can be created to provide cluster-wide defaults for various aspects of Consul. Every configuration
+entry has at least two fields: `Kind` and `Name`. Those two fields are used to uniquely identify a configuration entry.
+When put into configuration files, configuration entries can be specified as HCL or JSON objects.
+
+Example:
+
+```hcl
+Kind = "<supported kind>"
+Name = "<name of entry>"
+```
+
+Other `Kind` specific fields will be detailed below.
+
+## Usage
```

Comment:
I might put this at the end. I think it could flow nicely from, "Config entries provide defaults for aspects of Consul" to "currently the supported Config entry types are for services and proxies" 

Within each config entry type you already have some examples, so ending with usage with a bigger example would be good.